webapp/channels/src/utils/utils.tsx

```diff
@@ -1643,8 +1642,8 @@ export function getTrackFlowRole() {
     return trackFlowRole;
 }
 
-export function getRoleForTrackFlow() {
-    const startedByRole = TrackFlowRoles[getTrackFlowRole()];
+export function getRoleForTrackFlow(state: GlobalState) {
+    const startedByRole = TrackFlowRoles[getTrackFlowRole(state)];
 
     return {started_by_role: startedByRole};
 }
```

Comment:
Using this function as a selector will return always a new object. That sounds bad, but not sure if it is a problem right now.

Reply:
Good catch. It should use `createSelector` because of that